src/socket.rs

```diff
@@ -1500,7 +1500,11 @@ impl Socket {
         target_os = "tvos",
         target_os = "windows"
     ))]
-    pub fn set_tcp_fastopen(&self, value: u32) -> io::Result<()> {
+    pub fn set_tcp_fastopen(&self, mut value: u32) -> io::Result<()> {
+        if !cfg!(any(target_os = "linux", target_os = "android")) && value > 1 {
```

Comment:
We don't use `cfg!`
```suggestion
        #[cfg(not(any(target_os = "linux", target_os = "android")))]
        if value > 1 {
```

If that doesn't work it will need block around it.